components/visualization/src/main/scala/org/datacleaner/visualization/DensityAnalyzerChartScriptHeadElement.scala

```diff
@@ -20,22 +25,26 @@ class DensityAnalyzerChartScriptHeadElement(result: DensityAnalyzerResult, eleme
   override def toHtml(context: HtmlRenderingContext): String = {
 
     val annotations = result.getRowAnnotations
```

Comment:
If you `import collection.JavaConversions._` and define this like val annotations: `Map[(Int, Int) = result.getRowAnnotations`, you should be able to leave most of the code as it was.

http://www.scala-lang.org/docu/files/collections-api/collections_46.html